src/zui/components/ZUITagChip.tsx/getTagContrastColor.ts

```diff
@@ -0,0 +1,19 @@
+import { getContrastColor } from 'utils/colorUtils';
+import { funSwatches } from 'zui/theme/palette';
+
+export default function getTagContrastColor(color: string) {
+  let contrastColor = '';
+
+  Object.keys(funSwatches).forEach((funSwatch) => {
+    const colorObject = funSwatches[funSwatch as keyof typeof funSwatches];
```

Comment:
You have to typecast on line 8, because `funSwatch` is a string returned from `Object.keys()` and it's not typed any more specifically than `string`.

But you don't actually need the `funSwatch` key as far as I can tell, other than to extract the value. So maybe you could use `Object.values()` instead and avoid the typecast?

Reply:
Good point, i'll try this approach!